staging/src/k8s.io/apiserver/pkg/server/options/feature.go

```diff
@@ -33,17 +32,13 @@ func NewFeatureOptions() *FeatureOptions {
 	defaults := server.NewConfig()
 
 	return &FeatureOptions{
-		EnableGarbageCollection:   defaults.EnableGarbageCollection,
 		EnableProfiling:           defaults.EnableProfiling,
 		EnableContentionProfiling: defaults.EnableContentionProfiling,
 		EnableSwaggerUI:           defaults.EnableSwaggerUI,
 	}
 }
 
 func (o *FeatureOptions) AddFlags(fs *pflag.FlagSet) {
-	fs.BoolVar(&o.EnableGarbageCollection, "enable-garbage-collector", o.EnableGarbageCollection, ""+
```

Comment:
oh fun, we had that in k8s.io/apiserver before. To get the right understanding of our layering: the gc controller, how dependent is that on kube? Not at all?